Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Scripts/Input/Controllers/OculusHand.cs

```diff
@@ -283,15 +278,10 @@ public void UpdateController(OVRHand hand, OVRSkeleton ovrSkeleton, Transform tr
             }
         }
 
-#if LATER
         private void UpdateTeleport()
         {
-            if (MRTKOculusConfig.Instance.ActiveTeleportPointerMode == MRTKOculusConfig.TeleportPointerMode.None) return;
-
             MixedRealityInputAction teleportAction = MixedRealityInputAction.None;
 
-            IMixedRealityTeleportPointer teleportPointer = TeleportPointer;
-
             // Check if we're focus locked or near something interactive to avoid teleporting unintentionally.
```

Comment:
Do we need to do all this manual pointer management now that we're only using the "official" teleport pointer? This feels like a case that should be handled by the pointer mediator.

Reply:
Oh and pinging this one to see what you think

Reply:
Digging in a bit, actually yeah let's leave this as-is for now. We can migrate this logic to the DefaultPointerMediator when we do the work to make this logic work across all articulated hands (instead of just Oculus)

Reply:
This was there originally for the custom pointer, but given that the mediator wasn't really built with the teleporter in mind, I think this logic remains necessary. All aboard the train to eventually move it to the DefaultMediator